app/src/main/java/com/globant/practice/presentation/view/adapter/ListHomeAdapter.java

```diff
@@ -0,0 +1,117 @@
+package com.globant.practice.presentation.view.adapter;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.globant.practice.R;
+import com.globant.practice.domain.model.User;
+import java.util.List;
+
+/**
+ * Adapter of the RecyclerView on HomeActivity
+ * Created by jonathan.vargas on 17/04/2017.
+ */
+public class ListHomeAdapter extends RecyclerView.Adapter<ListHomeAdapter.ListHomeViewHolder>
```

Comment:
This name does not say to much. Perhaps a more descriptive name like _SubscriberAdapter_ could work better with the purpose of the adapter.